<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>jeux</class>
 <widget class="QWidget" name="jeux">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>jeux</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/img/fondConnection.png);</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>841</width>
     <height>611</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="lineEditNomJoueur">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Z003</family>
      <pointsize>25</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMdp">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Z003</family>
      <pointsize>25</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="labelPseudo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <italic>true</italic>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Votre Pseudo</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMdp">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <italic>true</italic>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>MDP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLancer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>271</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Vamos De Fiesta</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelIP">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <italic>true</italic>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>IP du Serveur</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditIP">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>390</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Z003</family>
      <pointsize>25</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
